src/llm-eval/README.md

```diff
@@ -0,0 +1,297 @@
+# Evaluating a LLM with .NET
+
+This repo contains a library and samples for evaluating the output of an LLM, similar to what we  uses in a RAG architecture. There are many parameters that affect the quality and style of answers generated by a chat app, such as the system prompt, search parameters, and GPT model parameters.
+
+This repository offers tools and samples to make it easier to run evaluations on Large Language Models.
+
+Table of contents:
+
+* [Setting up this project](#setting-up-this-project)
+* [Models to test, to evaluate and to generate data](#Models-to-test,-to-evaluate-and-to-generate-data)
+* [Generating ground truth data](#Generate-a-QA-from-a-Topic)
+* [Running an evaluation](#running-an-evaluation)
+* [Viewing the results](#viewing-the-results)
+
+The sample console application includes several uses cases
+
+- Generate and eval a QA using a LLM
+- Eval 2 harcoded QAs
+- Eval a harcoded User Story
+- Eval a List of User Stories loaded from a file
+- Generate and Eval a list of QAs using a LLM
+- Type a topic to generate a QA and eval the QA
+- Type the QA to be evaluated
```

Comment:
```suggestion
- Generate and evaluate a QAs using a LLM
- Evaluate 2 harcoded QAs
- Evaluate a harcoded User Story
- Batch evaluate a list of User Stories loaded from a file
- Generate and evaluate a list of QAs using a LLM
- Manually generate a QA and eval the QA
- Manually evaluate the QA 
```

Reply:
Consider spelling out (i.e. Question-Answer Pairs or Questions and Answers (QA))